awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupDetail/InventoryGroupDetail.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import { t } from '@lingui/macro';
+
+import { CardBody, Button } from '@patternfly/react-core';
+import { withI18n } from '@lingui/react';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { VariablesInput } from '@components/CodeMirrorInput';
+import ContentError from '@components/ContentError';
+import AlertModal from '@components/AlertModal';
+import { formatDateString } from '@util/dates';
+
+import { GroupsAPI } from '@api';
+import { DetailList, Detail } from '@components/DetailList';
+
+const ActionButtonWrapper = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 20px;
+  & > :not(:first-child) {
+    margin-left: 20px;
+  }
+`;
+function InventoryGroupDetail({ i18n, history, match, inventoryGroup }) {
+  const [error, setError] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  const handleDelete = async () => {
+    try {
+      await GroupsAPI.destroy(inventoryGroup.id);
+      history.push(`/inventories/inventory/${match.params.id}/groups`);
+    } catch (err) {
+      setError(err);
+    }
+  };
+  if (error) {
+    return <ContentError />;
```

Comment:
We should show an AlertModal for deletion errors. < ContentError > is reserved for Content Loading errors [(fetching data to initialize a page or populate a list)](https://github.com/ansible/awx/blob/devel/awx/ui_next/CONTRIBUTING.md#handling-api-errors).